app/views/compute_resources/form/_ovirt.html.erb

```diff
@@ -1,3 +1,13 @@
+<div class="clearfix">
+  <div class="form-group">
+    <div class="col-md-6">
+      <div class='alert alert-warning'>
+        <%= _('oVirt integration is deprecated, more information') %>
+        <%= link_to(_('here'), 'https://access.redhat.com/announcements/6960518', target: '_blank') %>.
```

Comment:
It's odd that this mentions oVirt in the header but then links to the RHV deprecation. https://ovirt.org/ doesn't mention anything about deprecation.

Also, this is not the proper way to translate. If there's a language where `here` isn't the last word, the sentence becomes weird. You should interpolate, though that's always tricky with links.

Perhaps split up the lines like this:
```suggestion
        <%= _('oVirt integration is deprecated.') %>
        <%= link_to(_('More information.'), 'https://access.redhat.com/announcements/6960518', target: '_blank') %>.
```

Reply:
Alert updated.

> It's odd that this mentions oVirt in the header but then links to the RHV deprecation.

I agree but except the red hat announcement there is nothing I can link to, no other source of the information.